tests/scripts/thread-cert/v1_2_LowPower_5_3_01_SSEDAttachment.py

```diff
@@ -0,0 +1,149 @@
+
+import unittest
+
+from config import ADDRESS_TYPE
+from mac802154 import MacHeader
+from pktverify import consts
+from pktverify.packet_verifier import PacketVerifier
+
+import thread_cert
+
+LEADER = 1
+ROUTER = 2
+SSED_1 = 3
+
+
+class LowPower_5_3_01_SSEDAttachment(thread_cert.TestCase):
+    TOPOLOGY = {
+        LEADER: {
+            'version': '1.2',
+            'name': 'LEADER',
+            'mode': 'rsdn',
+            'panid': 0xface,
+            'allowlist': [ROUTER, SSED_1]
+        },
+        ROUTER: {
+            'version': '1.2',
+            'name': 'ROUTER',
+            'mode': 'rsdn',
+            'panid': 0xface,
+            'router_selection_jitter': 1,
+            'allowlist': [LEADER],
+        },
+        SSED_1: {
+            'version': '1.2',
+            'name': 'SSED_1',
+            'mode': 's',
+            'panid': 0xface,
+            'allowlist': [LEADER],
+        },
+    }
+
+    def test(self):
+        self.nodes[LEADER].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+
+        self.nodes[ROUTER].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[ROUTER].get_state(), 'router')
+
+        self.nodes[SSED_1].set_csl_period(consts.CSL_DEFAULT_PERIOD)
+        self.nodes[SSED_1].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[SSED_1].get_state(), 'child')
+
+        self.simulator.go(5)
+
+        self.collect_rloc16s()
+        self.collect_rlocs()
+
+        # Verify connectivity by sending an ICMP Echo Request from the Router to SSED_1 mesh-local address via the DUT
+        # Set ping timeout as two CSL periods.
+        timeout = 2 * consts.CSL_DEFAULT_PERIOD * consts.US_PER_TEN_SYMBOLS / 1000000
+        self.assertTrue(self.nodes[ROUTER].ping(self.nodes[SSED_1].get_ip6_address(ADDRESS_TYPE.RLOC),
+                                                timeout=timeout))
+
+        # Verify fragmented CSL transmission
+        self.assertTrue(self.nodes[ROUTER].ping(self.nodes[SSED_1].get_ip6_address(ADDRESS_TYPE.RLOC),
+                                                size=128,
+                                                timeout=timeout))
+
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+        LEADER = pv.vars['LEADER']
+        ROUTER = pv.vars['ROUTER']
+        SSED_1 = pv.vars['SSED_1']
+
+        # Step 1: ensure the Leader is sending MLE Advertisements
+        pkts.filter_wpan_src64(LEADER) \
+            .filter_mle_cmd(consts.MLE_ADVERTISEMENT) \
+            .must_next()
+
+        # Step 2: Verify that the frame version is IEEE Std 802.15.4-2015 of the MLE Child Update Response from the
+        # DUT to SSED_1
+        pkts.filter_wpan_src64(LEADER) \
+            .filter_wpan_dst64(SSED_1) \
+            .filter_mle_cmd(consts.MLE_CHILD_UPDATE_RESPONSE) \
+            .filter_wpan_version(consts.MAC_FRAME_VERSION_2015) \
+            .must_next()
```

Comment:
> A Parent that receives a Child Update Request which includes the CSL Synchronized Timeout TLV SHOULD include the CSL Accuracy TLV in the response

Better to add check on the existence of CSL Accuracy TLV, if it is not in Test Plan, better to file an issue

Reply:
Not in Test Plan yet, and currently not in our implmentation. Will file an issue later.